articles/active-directory/cloud-infrastructure-entitlement-management/onboard-gcp.md

```diff
@@ -65,37 +65,39 @@ There are several moving parts across GCP and Azure, which are required to be co
     Optionally, specify **G-Suite IDP Secret Name** and **G-Suite IDP User Email** to enable G-Suite integration.
 
 
-1. You can either download and run the script at this point or you can do it in the Google Cloud Shell.
+1. You can either download and run the script at this point or you can run it in the Google Cloud Shell.
 
 1. Select **Next** after successfully running the setup script. 
 
 Choose from three options to manage GCP projects. 
 
 #### Option 1: Automatically manage 
 
-The automatically manage option allows projects to be automatically detected and monitored without extra configuration. Steps to detect list of projects and onboard for collection:  
+The automatically manage option allows projects to be automatically detected and monitored without extra configuration. Requirements to detect list of projects and onboard for collection:  
 
-1. Firstly, grant **Viewer** and **Security Reviewer** role to service account created in previous step at organization, folder or project scope. 
+1. In GCP, you will need to grant **Viewer** and **Security Reviewer** role to service account created in previous step at a project, folder or organization. 
 
-To enable controller mode 'On' for any projects, add following roles to the specific projects:
-- Role Administrators
-- Security Admin 
+ > [!NOTE]
+ > To enable controller mode 'On' for any projects, add following roles to the specific projects:
+ > - Role Administrators
+ > - Security Admin 
 
-2. Once done, the steps are listed in the screen, which shows how to further configure in the GPC console, or programmatically with the gCloud CLI.
+2. The requred commands to be run in Google Cloud Shell are listed in the Manage Authorization screen for each scope of a project, folder or organization. This can also be configure in the GPC console.
```

Comment:
This doesn't seem like a step. Do we want the customer to View the commands to run, and then select the commands they need to run? If this isn't a step and is just information, this can be a sentence after Security Admin in the previous step that's providing them with this info. If it is a step, then I suggest what I have below, but we need to also clarify what they need to do? Do they need to select a command from the available commands? 

Available commands to run in Google Cloud Shell are listed in the **Manage Authorization** screen for each project, folder or organization. This is also configured in the GCP console.